include/interface/libs/IMUKit.h

```diff
@@ -0,0 +1,32 @@
+// Leka - LekaOS
+// Copyright 2023 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <functional>
+namespace leka::interface {
+
+struct EulerAngles {
+	float pitch;
+	float roll;
+	float yaw;
+};
+
+class IMUKit
+{
+  public:
+	using angles_ready_callback_t = std::function<void(const EulerAngles &)>;
+
+	virtual ~IMUKit() = default;
+
+	virtual void init()	 = 0;
```

Comment:
not sure we need an `init()` -- better keep it to the minimal, MotionKit should not depend on `init()`